src/app/app.module.ts

```diff
@@ -5,26 +5,25 @@ import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { ConfigComponent } from './config/config.component';
 import { DisplayCurrentWeatherComponent } from './display-current-weather/display-current-weather.component';
-import { NgbModule, NgbTypeahead } from '@ng-bootstrap/ng-bootstrap';
 import { CitySearchComponent } from './city-search/city-search.component';
 import { CommonModule } from '@angular/common';
 import { FormsModule } from '@angular/forms';
 import { HttpClient, HttpClientModule } from '@angular/common/http';
+import { CitySearchModule } from './city-search/city-search.module';
 
 
 @NgModule({
   declarations: [
     AppComponent,
     ConfigComponent,
     DisplayCurrentWeatherComponent,
-    CitySearchComponent
```

Comment:
Replaced `CitySearchComponent` declaration with an import of existing `CitySearchModule`